src/drivers/driver_zephyr.c

```diff
@@ -1356,6 +1356,42 @@ static int nl80211_get_ext_capab(void *priv, enum wpa_driver_if_type type,
 	return 0;
 }
 
+static int wpa_drv_zep_get_conn_info(void *priv, struct wpa_conn_info *ci)
+{
+	struct zep_drv_if_ctx *if_ctx = NULL;
+	const struct zep_wpa_supp_dev_ops *dev_ops = NULL;
+	int ret = -1;
+
+	if (!priv) {
+		wpa_printf(MSG_ERROR, "%s: Invalid handle\n", __func__);
+		goto out;
+	}
+
+	if (!ci) {
+		wpa_printf(MSG_ERROR, "%s: Invalid params\n", __func__);
+		goto out;
+	}
+
+	if_ctx = priv;
+	dev_ops = if_ctx->dev_ctx->config;
+
+	os_memset(ci, 0, sizeof(*ci));
```

Comment:
Probably this memset is not required. As it has been set at calling function.

Reply:
removed the memset call.